### ISSUE TITLE
[NVC] NovaCustom V540TU 14th Gen Dasharo Release v0.9.0 issues

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#955]
### Component

other

### Device

NovaCustom V54 14th Gen

### Dasharo version

v0.9.0

### Brief summary

Newsletter has numerous bugs, which should be eliminated by spell checkers and automatic tooling.

### How reproducible

Always

### How to reproduce

Read newsletter

### Expected behavior

No concerns about content quality.

### Actual behavior

Quality issues:
- [ ] `We're excited to let you know, that Dasharo v0.9.0 firmware update for NovaCustom V540TU 14th Gen has just released!` does not sound like proper English -> Maybe it should be `We're excited to let you know that the Dasharo v0.9.0 firmware update for NovaCustom V540TU 14th Gen has just been released!` ?
- [ ] `Please use Dasharo Tools Suite from this link to update your firmware: https://github.com/Dasharo/meta-dts/releases/tag/v2.0.0-rc3` - maybe it would be more user friendly to really make `this link` a link instead of bare text in HTML email?
- [ ] `For those intrigued by the mechanics, click here to access the comprehensive documentation explaining what the Dasharo Openness Score is, the rules it follows, and how reports are generated.` - `here` is not a link
- [ ] Openness Score value in its current form is questionable - this topic was already discussed e.g. [here](https://github.com/Dasharo/dasharo-issues/issues/917) results of that discussion is [here](https://github.com/Dasharo/dasharo-issues/issues/924)
- [ ] Feature request: We provided links to all components' licenses at some point. I think that information should be included in SBOM's release notes. It has already happened a couple of times when someone asked about licenses for all components included. Maybe Opness Score should also account for that somehow.

**Comment by BeataZdunczyk** [comment]
> Please use Dasharo Tools Suite from this link to update your firmware: https://github.com/Dasharo/meta-dts/releases/tag/v2.0.0-rc3 - maybe it would be more user friendly to really make this link a link instead of bare text in HTML email?

True, we couldn't make it clicable in post editing. It is not something we want to incorporate into the template. It was an exception.

Scheduling fixing of the 1st, 3rd, and 5th points from the checklist. Thank you for the review.

We will address the 4th point in this issue: https://github.com/Dasharo/dasharo-issues/issues/924

**Comment by pietrushnic** [comment]
> True, we couldn't make it clickable in post editing. It is not something we want to incorporate into the template. It was an exception.

The way you address that is manual correction of HTML before importing to Mailtrain.

**Comment by philipandag** [comment]
Addressing points 1, 3 and 5:
https://git.3mdeb.com/3mdeb/dasharo-templates/pulls/4

**Comment by philipandag** [comment]
Point 5 has been addressed in https://git.3mdeb.com/3mdeb/dasharo-templates/pulls/4.
The `artifacts/.../relnotes.md` file now includes links to the components's licenses.

**Comment by pietrushnic** [comment]
@philipandag, a public repo with private link references. I'm pretty sure not everyone would like that. My suggestion would be to resolve this based on the next public release, which will prove the issue no longer exists.

**Comment by philipandag** [comment]
@pietrushnic Oh, I didn't notice that. Registering at `git.3mdeb.com` does require an email in `3mdeb.com` domain, which I have only just checked, so the repo is not pubilc. Should I modify or delete the previous comments?

**Comment by pietrushnic** [comment]
@philipandag No, the Internet will remember you forever, no matter what you do. If you delete it, you may bring more attention than needed. Leave it as it is, and do not allow it to happen again.
